reference/configuration/framework.rst

```diff
@@ -858,10 +859,136 @@ setting the value to ``null``:
 templating
 ~~~~~~~~~~
 
-.. _reference-framework-assets-version:
+hinclude_default_template
```

Comment:
Yes, it is related to the `hinclude` javascript library :
http://mnot.github.io/hinclude/